src/components/Dashboard/EventCard.vue

```diff
@@ -114,6 +113,24 @@ const hasCall = computed(() => {
 		&& props.eventRoom.start * 1000 <= (Date.now() - 600_000) // 10 minutes buffer
 })
 
+const attachmentInfo = computed(() => {
+	if (hasAttachments.value) {
+		const file = Object.entries(props.eventRoom.eventAttachments)[0]?.[1]
+		const totalAttachments = Object.keys(props.eventRoom.eventAttachments).length
```

Comment:
I think it makes now more sense to replace `hasAttachments` computed with
```js
const totalAttachments = computed(() => Object.values(props.eventRoom.eventAttachments))
```
Should be enough for anything (length, first file, extra label)